records/2023-11-24.system-status.md

```diff
@@ -83,18 +83,12 @@ The API Lambda will interact with 2 different systems:
 We will write all the above results into a singular .json and commit it to github.
 We should use multi-processing so all endpoints are fired up and run in parallel vs sequence.
 
-### Github Pages
-Github can host static webpages.
-The github repo will display status.notification.canada.ca
+### Front-end site
+We will use s3+cloudfront to serve up both staging and production versions of the site (`status.staging.notification.cdssandbox.xyz` and `status.notification.canada.ca`)
 
 #### The final flow will look like
 
-API Lambda -> Github Repo -> Static Page
-
-### Considerations
```

Comment:
we should leave this consideration - we are still deploying within AWS and have the issue of being on the same cloud provider having issues... aka we have increased our risk

Reply:
AWS Wide outages are very rare, more like a single region would go down, so cross region replication would be probably a good enough mitigation for that issue. 

Just using CloudFront should be enough, as it does edge caching so it should be very low risk of it going down.

If you really want a backup we can always deploy to azure and host the static site there instead or as well so we have a backup.

Reply:
If you are curious here is a list of all the major outages AWS has had since it's inception, s3 going down is 100% considered a major outage due to how important it is to AWS

https://awsmaniac.com/aws-outages/

Reply:
Yeah I think this covers us plenty, much better than our GH pages plan!